core/services/directrequestocr/drlistener.go

```diff
@@ -233,6 +296,8 @@ func (l *DRListener) handleOracleRequest(request *ocr2dr_oracle.OCR2DROracleOrac
 		"data", fmt.Sprintf("%0x", request.Data),
 	)
 
+	promRequestDataSize.WithLabelValues(l.oracleHexAddr).Add(float64(len(request.Data)))
```

Comment:
Does Add() mean that we will accumulate the sum of all request sizes so far? I'm not sure if that's very useful.

Reply:
I changed that to Histogram metric type now. Similar to what VRF used for data sizes and for timings.